eos/parameters/sm-and-eft.yaml

```diff
@@ -7,45 +7,52 @@ groups:
   - title: 'Standard Model Parameters'
     description: ''
     parameters:
-      'hbar' :
+      'QM::hbar' :
           central: +6.58211928e-25
           min:     +6.58211913e-25
           max:     +6.58211943e-25
+          latex:    '$\hbar$'
           comments: 'in GeV s, cf. [PDG2012]:  p. 4'
       # Factorization scale
       'mu' :
           central: +4.2
           min:     +2.4
           max:     +9.6
+          latex:    '$\mu$'
           comments: ''
       # GSW Parameter
       'GSW::sin^2(theta)' :
           central: +0.23116
           min:     +0.23104
           max:     +0.23128
+          latex:    '$\sin(\theta)^2_{\text{GSW}}$'
```

Comment:
I would write `$\sin(\theta_{\text{GSW}})^2$`

Reply:
Can we leave the superscript attached to the sin, rather than to its argument, as in the usual mathematical notation?

Reply:
Sure, I focused on the subscript but I fully agree with the `sin^2`